announcements/2021/week08.md

```diff
@@ -0,0 +1,20 @@
+# :zap: This week in provider-independent announcements (February 26, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[Prometheus Operator v0.7.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.7.0/) upgrades to upstream chart v13.10.0, which contains Prometheus v2.24.0. (:warning: This is a breaking change and requires Helm 3 to install.)
```

Comment:
@giantswarm/team-halo What is the impact for the user that it's a breaking change? If they upgrade frm a prev. version, what will happen?

Reply:
The user should have no direct impact by the breaking changes the new version introduced because it is only updating helm apiVersion from v1 to v2. It means users should have helm 3 interface if they want to install this version.

Reply:
@tomahawk28 will take a look

Reply:
ty! What if they want to upgrade?